tests/test_detection_pytest.py

```diff
@@ -0,0 +1,129 @@
+         "medRssi": 70},
+    ]
+
+    # good data, he met two person twice
+    good_data3 = [
+        {"buid": "foo", "timestampStart": timestamp_ms_n_one_day_ago, "timestampEnd": 1, "maxRssi": 5, "avgRssi": 15,
+         "medRssi": 12},
+        {"buid": "foo", "timestampStart": timestamp_ms_n_one_day_ago2, "timestampEnd": 1, "maxRssi": 5, "avgRssi": 15,
+         "medRssi": 90},
+        {"buid": "baz", "timestampStart": timestamp_ms_n_one_day_ago, "timestampEnd": 1, "maxRssi": 5, "avgRssi": 15,
+         "medRssi": 80},
+        {"buid": "baz", "timestampStart": timestamp_ms_n_one_day_ago, "timestampEnd": 1, "maxRssi": 5, "avgRssi": 15,
+         "medRssi": 77},
+    ]
+
+    # good data, he met one person twice and one person also twice but once FIFTEEN_DAY ago
+    good_data4 = [
+        {"buid": "foo", "timestampStart": timestamp_date_no_days_to_count, "timestampEnd": 1, "maxRssi": 5,
+         "avgRssi": 15,
+         "medRssi": 12},
+        {"buid": "foo", "timestampStart": timestamp_ms_n_one_day_ago2, "timestampEnd": 1, "maxRssi": 5, "avgRssi": 15,
+         "medRssi": 90},
+        {"buid": "baz", "timestampStart": timestamp_ms_n_one_day_ago, "timestampEnd": 1, "maxRssi": 5, "avgRssi": 15,
+         "medRssi": 80},
+        {"buid": "baz", "timestampStart": timestamp_ms_n_one_day_ago, "timestampEnd": 1, "maxRssi": 5, "avgRssi": 15,
+         "medRssi": 77},
+    ]
+
+    # good data, he met one person twice and one person also twice but twice FIFTEEN_DAY ago
+    good_data5 = [
+        {"buid": "foo", "timestampStart": timestamp_date_no_days_to_count, "timestampEnd": 1, "maxRssi": 5,
+         "avgRssi": 15,
+         "medRssi": 12},
+        {"buid": "foo", "timestampStart": timestamp_date_no_days_to_count, "timestampEnd": 1, "maxRssi": 5, "avgRssi": 15,
+         "medRssi": 90},
+        {"buid": "baz", "timestampStart": timestamp_ms_n_one_day_ago, "timestampEnd": 1, "maxRssi": 5, "avgRssi": 15,
+         "medRssi": 80},
+        {"buid": "baz", "timestampStart": timestamp_ms_n_one_day_ago, "timestampEnd": 1, "maxRssi": 5, "avgRssi": 15,
+         "medRssi": 77},
+    ]
+
+    scores_good_data = {'foo': {'buid': 'foo', 'last_timestamp': timestamp_ms_n_one_day_ago, 'score': 35}}
+
+    scores_good_data2 = {'foo': {'buid': 'foo', 'last_timestamp': timestamp_ms_n_one_day_ago, 'score': 70}}
+
+    scores_good_data3 = {'foo': {'buid': 'foo', 'last_timestamp': timestamp_ms_n_one_day_ago2, 'score': 40},
+                         'baz': {'buid': 'baz', 'last_timestamp': timestamp_ms_n_one_day_ago, 'score': 34}}
+
+    scores_good_data4 = {'foo': {'buid': 'foo', 'last_timestamp': timestamp_ms_n_one_day_ago2, 'score': 5},
+                         'baz': {'buid': 'baz', 'last_timestamp': timestamp_ms_n_one_day_ago, 'score': 34}}
+
+    scores_good_data5 = {'baz': {'buid': 'baz', 'last_timestamp': timestamp_ms_n_one_day_ago, 'score': 34}}
+
+    assert detect(bad_data, datetime.now()) == scores_bad_data
```

Comment:
I'd prefer to have asserts near the data you test, not together at the end. The rest is fine, test cases are good separated from each other by comment lines.

Reply:
thanks